app/models/orders_package.rb

```diff
@@ -71,7 +72,7 @@ def undispatch_orders_package
   end
 
   def update_state_to_designated
-    package.update_allow_web_publish_to_false
+    package.unpublish
```

Comment:
Much clearer!

Reply:
Yup.. its much better and clearer now. Thanks.